stuffer/s2n_stuffer_text.c

```diff
@@ -110,8 +110,9 @@ int s2n_stuffer_skip_to_char(struct s2n_stuffer *stuffer, const char target)
 }
 
 /* Skips an expected character in the stuffer between min and max times */
-int s2n_stuffer_skip_expected_char(struct s2n_stuffer *stuffer, const char expected, int min, int max)
+int s2n_stuffer_skip_expected_char(struct s2n_stuffer *stuffer, const char expected, const int min, const int max)
```

Comment:
Change the signature to return `S2N_RESULT`

Reply:
and have an `uint32_t *num_bytes_skipped`

Reply:
Done.